cpu/stm32/include/can_params.h

```diff
@@ -43,11 +43,13 @@ static const can_conf_t candev_conf[] = {
         .rcc_mask = RCC_APB1ENR1_CAN1EN,
 #else
         .rcc_mask = RCC_APB1ENR_CAN1EN,
+#if CANDEV_STM32_CHAN_NUMOF > 1
```

Comment:
I don't quite understand this ifdef, isn't this the configuration of a single can device? Why is this if needed?

Reply:
> I don't quite understand this ifdef, isn't this the configuration of a single can device? Why is this if needed?

This is where the compile error happened. 
can_master, etc are for 2 or more CAN interfaces device. stm32f722 does not have this.
If look at candev_stm32.h https://github.com/RIOT-OS/RIOT/blob/0f7f1fdd982a4e66684d4d16fd57a79994f9ea86/cpu/stm32/include/candev_stm32.h#L104

Reply:
Ah thanks perfect!